draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -99,8 +99,8 @@
 
     <t>
       Most often, this is desirable when the data conveyed in a request is
-      too voluminous to be encoded into the request's URI. For example, while
-      this is an common and interoperable query:
+      too voluminous to be encoded into the request's URI. For example,
+      this is a common and interoperable query:
```

Comment:
```suggestion
      this is a common query pattern:
```

No need to mention "interoperable".  Interoperability at this level is not especially meaningful.  Yes, any resource that is identified thus can likely handle this request, but what else about it is interoperable?